code/game/objects/items/weapons/tools/simple_weapons.dm

```diff
@@ -200,3 +200,24 @@
 	max_upgrades = 2
 	tool_qualities = list(QUALITY_HAMMERING = 5)
 	spawn_blacklisted = TRUE
+
+/obj/item/weapon/tool/spear //spear
+	icon = 'icons/obj/weapons.dmi'
+	icon_state = "spearglass0"
+	wielded_icon = "spearglass1"
+	name = "spear"
+	desc = "A spiky bit of material tied onto a metal pole with some wire. It's an insult to spears across the galaxy - but it can still do some nasty damage and has some decent armor-piercing capabilities. Spears like these are often seen in the hands of vagrants, muggers, or desperate militias. Due to this weapon - if you could call it that - being so long, you're able to attack enemies from up to a tile away."
+	force = WEAPON_FORCE_PAINFUL
```

Comment:
this... completely removes any difference between diamond, plasteel, glass spears. Are you sure you don't want to use old maths?...

Reply:
You cannot modify old maths. This why i need to change this kind of stuff.